presto-docs/src/main/sphinx/develop/connector.rst

```diff
@@ -0,0 +1,62 @@
+==========
+Connectors
+==========
+
+First, if you have not already looked at :doc:`/develop/spi-overview`, check there for
```

Comment:
Is this necessary? It seems reasonable to assume that people will read the "Overview" section first.